go2.lic

```diff
@@ -183,7 +183,14 @@ if XMLData.game =~ /^DR/
 
   # Iterate through the aggregated map wayto overrides from above and set new stringprocs
   wayto_overrides.each do | key, values |
-    Map.list[values['start_room'].to_i].wayto["#{values['end_room']}"] = StringProc.new("#{values['str_proc']}")
+    start_room_id = values['start_room'].to_i
+    end_room_id = values['end_room'].to_i
+    travel_command = StringProc.new("#{values['str_proc']}")
+    travel_time = 0.2
+
+    start_room = Map.list[start_room_id]
+    start_room.wayto["#{end_room_id}"] = travel_command
+    start_room.timeto["#{end_room_id}"] = travel_time
```

Comment:
The `wayto` defines _what_ command to perform to get from one room to the other.

The `timeto` defines _how long_ it takes to move between the rooms, and is required by the routing algorithm. If there's no `timeto` then the algorithm assumes there is no route, despite what `wayto` says.

Reply:
Are we better off adding timeto to our custom string procs, or hard coding .2?